terraform/modules/cloudwatch/alarms.tf

```diff
@@ -71,3 +71,137 @@ resource "aws_cloudwatch_metric_alarm" "load_balancer_request_spike" {
   }
 
 }
+
+/*
+An AWS account is placed under review if its SES bounce rate is ≥5%. Sending is automatically paused if its bounce rate is ≥10%. To avoid these outcomes, we send a warning alarm at 40% of the review threshold and a critical alarm at 80% of the review threshold. This provides early warning and margin for error.
+
+https://docs.aws.amazon.com/ses/latest/dg/reputationdashboardmessages.html#reputationdashboard-bounce
+*/
+
+# 5% * 40% = 2%. 5% * 80% = 4%.
+resource "aws_cloudwatch_metric_alarm" "ses_bounce_rate_warning" {
+  alarm_name        = "SES-BounceRate-Warning-AccountWide"
+  alarm_description = "Warning: The bounce rate for this AWS account has exceeded 2%. The Platform team should take action so AWS does not pause our ability to send email. Runbook link: https://github.com/cloud-gov/internal-docs/blob/main/docs/runbooks/Brokered-Services/AWS-SES.md"
+
+  metric_query {
+    id = "m1"
+    metric {
+      metric_name = "BounceRate"
+      namespace   = "AWS/SES"
+      period      = 300
+      stat        = "Average"
```

Comment:
question for all of these, I think:
is the `BounceRate` cumulative or bucketed? When AWS evaluates whether we're good senders, what window are they looking at?

(reading AWS's docs it sounds like the answer to the second question is "maybe")

Reply:
The best answer I can find is from this page, which says:

> We calculate your bounce rate based on a representative volume of email. A representative volume is an amount of email that represents your typical sending practices. To be fair to both high- and low-volume senders, the representative volume is different for each account and changes as the account's sending patterns change.

So it sounds like they sample delivery notifications. That doesn't explain if the thresholds are instantaneous or averaged over some period of time, though. Are there changes you'd like to see here if it's one or the other?

Reply:
if it's an average, we'd want to watch trends very closely. I think we have to assume this is windowed, and hope it's roughly the same window/metric they actually evaluate for blocks